### PR TITLE
Groovytestkit

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

File: liferay-workspace/tests/src/test/groovy/com/liferay/blade/tests/BladeTests.groovy [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+		def file = new File("${bladeclijar}")
+
+		if (!file.isFile()) {
+			file.newOutputStream()
+			file << URL(url).openStream()
+			file.close()
+		}
+
+		if (!file.isFile()) {
+			println "Test Failed"
```
Comment (lawrence-lee) [inline liferay-workspace/tests/src/test/groovy/com/liferay/blade/tests/BladeTests.groovy]: If it fails, should we have it throw an exception here? If so, what kind of exception?

File: liferay-workspace/tests/src/test/groovy/com/liferay/blade/tests/BladeTests.groovy [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+				}
+			}
+
+		then:
+			// check the list for error status
+
+			//result.errorList.isEmpty == true
+			assert(errorList.isEmpty());
+
+			result.task(":build").outcome == SUCCESS
```
Comment (lawrence-lee) [inline liferay-workspace/tests/src/test/groovy/com/liferay/blade/tests/BladeTests.groovy:117]: I'm not sure how this "then" section works. Do i use it to print out all of the errors it grabs from starting the bundles?

File: liferay-workspace/tests/src/test/groovy/com/liferay/blade/tests/BladeTests.groovy [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+			def errorList = []
+
+		when:
+			sampleBundles.each { sampleBundlefile ->
+				def installBundleOutput = executeBlade('sh', 'install', "file:${sampleBundlefile}").text()
+
+				def bundleID = installBundleOutput.substring(installBundleOutput.length() - 3)
+
+				if(installBundleOutput.contains("Failed") {
+					println "Test Failed"
```
Comment (lawrence-lee) [inline liferay-workspace/tests/src/test/groovy/com/liferay/blade/tests/BladeTests.groovy]: Throw an exception here as well? What do you think?

**Comment by gamerson** [comment]
hey @lawrence-lee can you rebase with the latest master and resend?  thanks!

**Pull request closed without merging**